src/components/Dropdown/index.jsx

```diff
@@ -0,0 +1,59 @@
+// @flow
+import React, { useState, useCallback } from "react";
+import * as _ from "lodash";
+
+type Props = {
+  initValue: $ReadOnly<{ value: string, label: string }>,
+  label: string,
+  items: $ReadOnlyArray<{ value: string, label: string }>
+};
+
+export default function Dropdown(props: Props) {
+  const { initValue, label, items } = props;
+  const [isExpanded, setIsExpanded] = useState(false);
+  const [selectedItem, setSelectedItem] = useState(initValue);
+  const expandMenu = useCallback(() => {
+    setIsExpanded(isShown => !isShown);
+  }, [setIsExpanded]);
+  const selectItem = useCallback(
+    item => {
+      setIsExpanded(false);
+      setSelectedItem(
+        _.find(items, function(o) {
+          return o.value === item.value;
+        })
+      );
+    },
+    [setIsExpanded, setSelectedItem]
+  );
+
+  return (
+    <div className="dropdown">
```

Comment:
All `dropdown` selectors should be renamed to `gc-dropdown` prefixed. `gc` stands for `graphen component`